ircd/s_user.c

```diff
@@ -491,9 +491,15 @@ register_local_user(struct Client *client_p, struct Client *source_p)
 
 	if(IsNeedSasl(aconf) && !*source_p->user->suser)
 	{
+
+		const char *sasl_only_client_message = ConfigFileEntry.sasl_only_client_message;
 		ServerStats.is_ref++;
 		sendto_one_notice(source_p, ":*** Notice -- You need to identify via SASL to use this server");
```

Comment:
I'm wondering if it would also be good to show the custom message here

Reply:
Done!